.goreleaser.yml

```diff
@@ -82,5 +82,5 @@ dockers:
 release:
   name_template: "v{{.Version}}"
   prerelease: auto # when we go to GA remove this line
-  draft: true # when we go to GA remove this line
+  draft: false # This needs to be false, or the container image will not be published
```

Comment:
This should result in a pre-release candidate since `prerelease: auto` is set and the tag has beta...?

Reply:
Yes, is that a problem?

Reply:
No problem, I think your fixing the leftover configurations when v2 wasn't on master.